privaterelay/settings.py

```diff
@@ -349,16 +349,7 @@
     ]
 
 
-# statsd middleware has to be first to catch errors in everything else
-def _get_initial_middleware() -> list[str]:
-    if STATSD_ENABLED:
-        return [
-            "privaterelay.middleware.ResponseMetrics",
-        ]
-    return []
-
-
-MIDDLEWARE = _get_initial_middleware()
+MIDDLEWARE = ["privaterelay.middleware.ResponseMetrics"]
```

Comment:
(suggestion, non-blocking) I agree with more simple logic on initial list of middlewares but not sure if the `ResponseMetrics` should be used in all environment. `STATSD_ENABLED` is `False` in local:
https://github.com/mozilla/fx-private-relay/blob/2d8973e9483bbaa876d46befda23bd9d8dbc1dde/.env-dist#L14-L15

```suggestion
if RELAY_CHANNEL == "local":
    MIDDLEWARE = []
else:
    MIDDLEWARE = ["privaterelay.middleware.ResponseMetrics"]
```

Reply:
I retained the spirit of the old logic by making the `ResponseMetrics` middleware do nothing if `STATSD_ENABLED` is False. This reminds me, I have a TODO to enable `STATSD_DEBUG` locally..